### PR TITLE
Added basic running instructions

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
@theute , could you please check/merge? This adds a RUNNING.adoc, with instructions on how to start with running Hawkular (ie: download, add user, set agent password, ...)

File: RUNNING.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+unzip -q "hawkular-services-dist-${HAWKULAR_RELEASE}.zip"
+
+cd "${HAWKULAR_HOME}"
+echo "Hawkular Home is: ${HAWKULAR_HOME}"
+"${HAWKULAR_HOME}/bin/add-user.sh" \
+  -a \
+  -u "${HAWKULAR_USERNAME}" \
+  -p "${HAWKULAR_PASSWORD}" \
+  -g read-write,read-only
+"${HAWKULAR_HOME}/bin/standalone.sh" -Dhawkular.rest.user="${HAWKULAR_USERNAME}" -Dhawkular.rest.password="${HAWKULAR_PASSWORD}"
```
Comment (theute) [inline RUNNING.adoc]: You should add -Dhawkular.agent.enabled=true or the agent won't run (and if the agent doesn't run then the other parameters are not useful).

**Comment by jpkrohling** [comment]
Added `-Dhawkular.agent.enabled=true`

**Comment by theute** [comment]
We can debate about having the script file as a script file outside a document, but beside this, LGTM

**Pull request merged**